library/src/main/java/com/vinted/coper/Coper.kt

```diff
@@ -52,10 +52,25 @@ interface Coper {
         onSuccess: suspend (PermissionResult.Granted) -> Unit
     )
 
+    suspend fun isRequestPendingSafe(): Boolean
+
+    /**
+     * @return true if all [permissions] is granted, false if at least one denied.
+     */
+    suspend fun isPermissionsGrantedSafe(vararg permissions: String): Boolean
```

Comment:
Is it possible to check which permissions were granted or which permissions were denied?